core/server/models/base/index.js

```diff
@@ -519,8 +528,8 @@ ghostBookshelf.Model = ghostBookshelf.Model.extend({
      * @param {Object} data Has keys representing the model's attributes/fields in the database.
      * @return {Object} The filtered results of the passed in data, containing only what's allowed in the schema.
      */
-    filterData: function filterData(data) {
-        var permittedAttributes = this.prototype.permittedAttributes(),
+    filterData: function filterData(data, {methodName, context} = {}) {
+        var permittedAttributes = this.prototype.permittedAttributes(methodName, context),
```

Comment:
Wanne keep the pattern of using `options`?

Reply:
sure :+1:

Reply:
Thanks 👍